src/utils.js

```diff
@@ -246,3 +252,31 @@ export const normalizeMonthlySynthData = sourceData => {
 		...dailyData,
 	];
 };
+
+export const calculateInterval = (period, displayName) => {
+	if (!displayName) {
+		return 1;
+	}
+	let interval = period === CHARTS.DAY ? 2 : 6;
+	if (displayName.startsWith('synths') && period === CHARTS.MONTH) {
+		interval = 2;
+	} else if (displayName.startsWith('synths') && period === CHARTS.WEEK) {
+		interval = 50;
+	} else if (displayName.startsWith('synths') && period === CHARTS.DAY) {
+		interval = 12;
+	} else if (displayName.startsWith('Snx') && period === CHARTS.DAY) {
```

Comment:
We should probably use constants for the display name. Not a big deal but I can see "synths" is all lowercase and 
"Snx" isn't. It's a bit confusing